src/main/java/core/basesyntax/BooleanExpression.java

```diff
@@ -13,6 +13,7 @@
      * <p>Это задание необходимо решать с использованием логических операторов.</p>
      */
     public boolean booleanExpression(boolean a, boolean b, boolean c, boolean d) {
-        return false;
+
```

Comment:
remove blank line

Reply:
i believe, i understand the solution, i  found info about XOR operator and how it works. In our task it will make a statement "true" only if one from two booleans is true and a second one is false. And that helps to simplify the solution and remove the description of all cases as i did before. 
would appreciate, if you let me know whether i understand it all correctly)